test/unit/io.js

```diff
@@ -861,6 +861,70 @@ describe('Input/output', function () {
     });
   });
 
+  it('TIFF lzw compression shrinks test file', function (done) {
+    const startSize = fs.statSync(fixtures.inputUncompressedTiff).size;
```

Comment:
The variable that you created is called `inputTiffUncompressed` not `inputUncompressedTiff`.

Please, please run the tests locally and ensure they are passing before pushing further code changes to Github.